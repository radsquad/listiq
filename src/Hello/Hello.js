import React from 'react'

class Hello extends React.Component {
  render () {
    return (
      <div>
        <span style={{color: 'cyan'}}>HelloComponent</span>
      </div>
    )
  }
}

export default Hello
