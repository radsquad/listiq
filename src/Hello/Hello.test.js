import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Hello from './Hello'

it('matches snapshot', function () {
  var component = shallow(<Hello />)
  var tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
