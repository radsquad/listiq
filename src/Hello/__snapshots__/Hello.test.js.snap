// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <span
    style={
      Object {
        "color": "cyan",
      }
    }
  >
    HelloComponent
  </span>
</div>
`;
